.github/workflows/main.yml

```diff
@@ -53,17 +62,23 @@ jobs:
   # Check Code style quickly by running `rustfmt` over all code
   rustfmt:
     runs-on: ubuntu-latest
+    if: "github.repository == 'rust-lang/cargo'"
     steps:
     - uses: actions/checkout@v4
+      with:
+        fetch-depth: 2
```

Comment:
I'm assuming depth of 2 is merge commit plus top commit from users branch? If we only need to merge commit, we should probably just use that.

Reply:
This is apparently useful for [checking `HEAD^`](https://github.com/actions/checkout#checkout-head). But looking more closely at the `checkout` action the default is already at `1`, so I'm gonna revert those.

Reply:
Done reverted.